python_http_client/exceptions.py

```diff
@@ -0,0 +1,53 @@
+class HTTPError(Exception):
+	''' Base of all other errors'''
+	def __init__(self,error):
+		self.code = error.code
+		self.reason = error.reason
+		#self.headers = error.headers
```

Comment:
The HTTPError thrown by `urllib` module doesn't have a body attribute and in python versions <= 3.3, they don't have the headers attribute as well. I just commented this out so that this can be discussed further.